### ISSUE TITLE
Showing both modules (Listing and administration) in one article

### ISSUE THREAD

**Issue by wilber-x** [ContaoDMS/dms#43]
this results in a blank page?

**Comment by cliffparnitzky** [comment]
Fehlermeldung:

````
[19-Mar-2015 21:15:11 Europe/Berlin] 
PHP Warning: filemtime(): stat failed for /www/htdocs/w011f1e1/html/files/protected/Austausch Gemeindebrief/Bilder GMZ/Au in /www/htdocs/w011f1e1/html/system/modules/core/library/Contao/File.php on line 236
#0 [internal function]: __error(2, 'filemtime(): st...', '/www/htdocs/w01...', 236, Array)
#1 /www/htdocs/w011f1e1/html/system/modules/core/library/Contao/File.php(236): filemtime('/www/htdocs/w01...')
#2 /www/htdocs/w011f1e1/html/system/modules/core/elements/ContentDownloads.php(227): Contao\File->__get('mtime')
#3 /www/htdocs/w011f1e1/html/system/modules/core/elements/ContentElement.php(150): Contao\ContentDownloads->compile()
#4 /www/htdocs/w011f1e1/html/system/modules/core/elements/ContentDownloads.php(91): Contao\ContentElement->generate()
#5 /www/htdocs/w011f1e1/html/system/modules/core/library/Contao/Controller.php(457): Contao\ContentDownloads->generate()
#6 /www/htdocs/w011f1e1/html/system/cache/dca/tl_content.php(1129): Contao\Controller->getContentElement('360')
#7 /www/htdocs/w011f1e1/html/system/modules/core/drivers/DC_Table.php(4153): tl_content->addCteType(Array)
#8 /www/htdocs/w011f1e1/html/system/modules/core/drivers/DC_Table.php(360): Contao\DC_Table->parentView()
#9 /www/htdocs/w011f1e1/html/system/modules/core/classes/Backend.php(425): Contao\DC_Table->showAll()
#10 /www/htdocs/w011f1e1/html/contao/main.php(137): Contao\Backend->getBackendModule('article')
#11 /www/htdocs/w011f1e1/html/contao/main.php(288): Main->run()
#12 {main}

[19-Mar-2015 21:15:15 Europe/Berlin] PHP Fatal error:  Cannot redeclare class ContaoDMS\DmsLoader in /www/htdocs/w011f1e1/html/system/modules/DocumentManagementSystem/classes/DmsLoader.php on line 429
````

**Comment by cliffparnitzky** [comment]
@wilber-x We could not reproduce the probleme. Using the listing and the management module both on one page works fine (tested on various haostings).

It seems to be a subsequent error. Have a look at the error message. The first part describes an error while using the CE "Download". Please check your page / article and remove such an element.

**Comment by wilber-x** [comment]
I have removed the download element,
the (first) errormessages diappeared, but the message 

[23-Mar-2015 21:02:36 Europe/Berlin] PHP Fatal error:  Cannot redeclare class ContaoDMS\DmsLoader in /www/htdocs/w011f1e1/html/system/modules/DocumentManagementSystem/classes/DmsLoader.php on line 429

 and the blank page still occurs.

**Comment by cliffparnitzky** [comment]
Hmpf, you found a terrible bug. This errors only occures, if the listing module comes at first and the management module at seconad position.

If the order is changed, no errors occure.

I will fix it now.

**Comment by cliffparnitzky** [comment]
Fixed in version 3.1.1
